lib/matplotlib/quiver.py

```diff
@@ -173,6 +174,13 @@
 
     'mid' is a synonym for 'middle'.
 
+head_pos : string, in {'tail', 'mid', 'middle', 'tip'},
+    or float, in range 0.0<head_pos<1.0
+    optional, default: 'tip'
```

Comment:
Type must fit on one line, range can be in description.
```suggestion
head_pos : {'tip', 'mid', 'middle', 'tail'}, or float, default: 'tip'
```